release/1.8.0: document environment naming conventions

climbfuji opened the pull request:
### Summary

Resolves #1291 by adding a table to `doc/source/PreConfiguredSites.rst`.

### Testing

Readthedocs build: https://spack-stack--1300.org.readthedocs.build/en/1300/PreConfiguredSites.html

### Applications affected

None

### Systems affected

None

### Dependencies

None

### Issue(s) addressed

Resolves #1291 

### Checklist
- [x] This PR addresses one issue/problem/enhancement, or has a very good reason for not doing so.
- [ ] ~~These changes have been tested on the affected systems and applications.~~
- [ ] ~~All dependency PRs/issues have been resolved and this PR can be merged.~~

climbfuji committed:
Update doc/source/PreConfiguredSites.rst with environment naming conventions

climbfuji committed:
[skip ci] Fix typos in newly added table

srherbener commented:
@climbfuji is this PR supposed to be based on `release/1.8.0` instead of `develop`?

climbfuji commented:
> @climbfuji is this PR supposed to be based on `release/1.8.0` instead of `develop`?

Yes :-) and now it is. Thanks for catching this!

RatkoVasic-NOAA committed:
Update Jet mirrors.yaml

RatkoVasic-NOAA committed:
Update Jet packages_intel.yaml

AlexanderRichert-NOAA commented on configs/sites/tier1/jet/mirrors.yaml:
```diff
@@ -1,15 +1,15 @@
       endpoint_url: null
     push:
-      url: file:///mnt/lfs4/HFIP/hfv3gfs/role.epic/spack-stack/source-cache
+      url: file:///contrib/spack-stack/mirror
```
My understanding is that space on /contrib is a bit more precious than lfs. Not that we're not allowed to put caches there, but from a being-good-neighbors perspective we might want to consider leaving the caches on lfs, and just have the installations themselves under /contrib.

AlexanderRichert-NOAA commented:
Documentation updates looks good, thanks for adding that!

climbfuji commented:
The jet updates have nothing to do with this PR and should be removed ...

RatkoVasic-NOAA committed:
Revert mirrors.yaml

RatkoVasic-NOAA committed:
Reverted packages_intel.yaml

RatkoVasic-NOAA commented:
> The jet updates have nothing to do with this PR and should be removed ...
@climbfuji , sorry, I saw other than document files in this PR. Reverted.

RatkoVasic-NOAA reviewed (approved):
I followed this convention on tier1 machines. Looks good.

climbfuji commented:
> > The jet updates have nothing to do with this PR and should be removed ...
> > @climbfuji , sorry, I saw other than document files in this PR. Reverted.

Yeah, apologies. This PR originally had `develop` as the target/base branch and showed all the changes between (release/1.8.0 plus this PR) and develop. Steve pointed that out and I changed the base branch to release/1.8.0. Let's do all the NOAA site config and doc updates in a separate PR from you, ok?
